citrix_hypervisor/assets/recommended_monitors/host_cpu_high.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "[Citrix Hypervisor] CPU load is high on VM {{citrix_hypervisor_host.name}} ",
```

```suggestion
	"name": "[Citrix Hypervisor] CPU load is high on host {{citrix_hypervisor_host.name}} ",
```
you mean host here instead of VM?